scripts/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='securedrop',
+    version='1.0',
+    packages=['securedrop', 'securedrop.tests'],
+    url='https://github.com/aokellermann/securedrop.git',
+    license='MIT License',
+    author='John Nay, Antony Kellermann, Robert Liani',
+    author_email='John_Nay@student.uml.edu',
```

Comment:
This should be my email (use the one listed on my github account).

Reply:
Changed.

Reply:
This should be aokellermann@gmail.com

Reply:
Changed